pkg/operator/controllers/controller_utils.go

```diff
@@ -71,6 +72,18 @@ func randString(n int) string {
 	return string(b)
 }
 
+// Determine whether to use an existing saved time variables or the current time for state duration
```

Comment:
method level comments in Go should start with method name, so this should be `getStateDuration determines whether ...`